Corona/Model/Change.swift

```diff
@@ -38,11 +38,11 @@ extension Change {
 
 	public var newConfirmedString: String { "+\(newConfirmed.groupingFormatted)" }
 	public var newRecoveredString: String { currentRecovered == 0 ? "-" : "+\(newRecovered.groupingFormatted)" }
-	public var newDeathsString: String { "+\(newDeaths.groupingFormatted)" }
+    public var newDeathsString: String { (lastDeaths < currentDeaths) ? "+\(newDeaths.groupingFormatted)" : "\(newDeaths.groupingFormatted)" }
```

Comment:
We just need to change the condition to the following:
`newDeaths > 0 ? "+\(newDeaths.groupingFormatted)" : "-"`